tests/test_state.py

```diff
@@ -175,6 +176,7 @@ def setUp(self):
         hs.config = default_config("tesths", True)
         hs.get_datastore.return_value = self.store
         hs.get_state_handler.return_value = None
+        hs.get_event_auth_handler.return_value = None
```

Comment:
I'm not sure why we need this change? It doesn't seem like any event auth code is touched in this PR?

Reply:
Strange, had test failures without this at one point but now is fine without